packages/frontend/web/server/htmlTemplate.ts

```diff
@@ -14,6 +16,7 @@ export const htmlTemplate = ({
     cssIDs,
     nonBlockingJS = '',
     fontFiles = [],
+    windowGuardianObject,
```

Comment:
Maybe call this `clientsideConfig` or replace the `Object` suffix with `Config` ?

Reply:
Done, mostly. I we now have `WindowGuardian` and `ClientSideConfig`.